qmri_mpm/sub-01/fmap/sub-01_magnitude1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Manufacturer": "SIEMENS ",
+  "ManufacturersModelName": "Prisma_fit",
+  "DeviceSerialNumber": "167025",
+  "StationName": "MRC35437",
+  "MagneticFieldStrength": 3,
+  "ScanningSequence": "GR",
+  "SequenceName": "*fm2d2r ",
+  "PulseSequenceDetails": "gre_field_mapping_1acq_rl ",
+  "RepetitionTimeExcitation": 1.02,
+  "EchoTime": 0.01,
+  "FlipAngle": 90,
+  "MTState": false,
+  "NumberShots": 1,
+  "PhaseEncodingDirection": "j",
+  "IntendedFor": [
+    "fmap/sub-anon_acq-headMTw_RB1COR.nii",
+    "fmap/sub-anon_acq-headPDw_RB1COR.nii",
+    "fmap/sub-anon_acq-headT1w_RB1COR.nii",
+    "fmap/sub-anon_acq-bodyMTw_RB1COR.nii",
+    "fmap/sub-anon_acq-bodyPDw_RB1COR.nii",
+    "fmap/sub-anon_acq-bodyT1w_RB1COR.nii"
```

Comment:
Is there a missing `sub-anon` that I just don't see, or is this an error?

https://github.com/bids-standard/bids-examples/pull/245/checks?check_run_id=1872898390

![image](https://user-images.githubusercontent.com/9084751/107541416-66113600-6bc7-11eb-90c7-1fb924e312b1.png)